manticore/manticore.py

```diff
@@ -168,10 +168,13 @@ def __init__(self, path_or_state, argv=None, workspace_url=None, policy='random'
         self.forward_events_from(self._executor)
 
         if isinstance(path_or_state, str):
-            assert os.path.isfile(path_or_state)
+            if not os.path.isfile(path_or_state):
+                raise Exception("The file %s doesn't exist" % path_or_state)
```

Comment:
can we please change the phrasing: "{} is not an existing regular file".format(path_or_state) ? it will be a little more accurate with the specification for `os.path.isfile`

i also think we should try to use the `str.format()` style of string interpolation; i think it is a little more modern :)